tests/psoc6/README

```diff
@@ -217,7 +217,7 @@ run-tests.py         : run tests from host platform
 
```

Comment:
Maybe at some point we could dynamically display how many and which tests passed/failed in the CI/CD flow. Similar to README badges that are available nowadays. Just an idea. @jaenrig-ifx @bargfred

Reply:
Not sure what you mean exactly, but let us discuss on the phone.

Reply:
![image](https://user-images.githubusercontent.com/105282041/221986817-4f967cd2-f88f-44bb-ad09-7580334871a2.png)

https://www.runforesight.com/blog/top-5-badges-that-will-show-your-github-repository-is-well-tested-trusted

Something similar to this.

Reply:
Yes, that is a good idea. That is already available for github actions. There is a badge in the readme for unix, but not for the rest of the ports.

Reply:
The diff is empty, not the file.

A badge would be nice.